### PR TITLE
Add modifications and corrections to labs examples

### PR DESCRIPTION
**Pull request by enriquebelarte**
Add modifications and corrections to lab examples.
Also add link to labs in main README.md.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -13,3 +13,5 @@ Install the bleeding edge KMMO in one command:
 ```shell
 oc apply -k https://github.com/rh-ecosystem-edge/kernel-module-management/config/default
 ```
+
+You can find examples on different use cases in this repo [labs](https://github.com/rh-ecosystem-edge/kernel-module-management/tree/main/docs/labs).
```
Comment (ybettan) [inline README.md:17]: I would suggest something bigger, like:

```suggestion

## Examples

You can find examples on different use cases in [labs](https://github.com/rh-ecosystem-edge/kernel-module-management/tree/main/docs/labs).
```

File: docs/labs/deviceplugin-kmm.yaml [hunk 2]
```diff
@@ -6,7 +6,7 @@ metadata:
 spec:
   devicePlugin:
     container:
-      image: "quay.io/myrepo/oc-dummy-device-plugin:0.1"
+      image: "quay.io/ebelarte/oc-dummy-device-plugin:0.1"
```
Comment (ybettan) [inline docs/labs/deviceplugin-kmm.yaml:9]: Can we continue using `myrepo` or even better `<org>`? Just to make sure it is clear to the user it needs to inject some input into the yaml.

File: docs/labs/README.md [hunk 3]
```diff
@@ -26,28 +26,55 @@ will be defined and created by the user. This definition will include:
...
+```console
+[enrique@ebelarte crds]$ oc get po
+NAME                                READY   STATUS      RESTARTS   AGE
+kmm-ci-a-hk472-1-build              0/1     Completed   0          106s
+kmm-ci-a-p4qx8-tmpch                1/1     Running     0          24s
+[enrique@ebelarte crds]$ oc exec -it kmm-ci-a-p4qx8-tmpch -- lsmod | grep kmm
+kmm_ci_a               16384  0
+[enrique@ebelarte crds]$ 
+```
+If you encounter issues when using KMM there are some other places where you can check.
```
Comment (qbarrand) [inline docs/labs/README.md:67]: Should we mention must-gather here?

Comment (enriquebelarte) [reply docs/labs/README.md:67]: Sure. Done.

**Pull request merged**